platform/darwin/src/MGLPolyline.mm

```diff
@@ -73,38 +73,34 @@ - (CLLocationCoordinate2D)coordinate {
                     return coordinates[i];
                 }
                 to = MGLRadianCoordinateFromLocationCoordinate(coordinates[i - 1]);
-                MGLRadianDirection direction = [self direction:from to:to] - 180;
+                CLLocationDirection direction = [self direction:from to:to] - 180;
                 MGLRadianCoordinate2D otherCoordinate = MGLRadianCoordinateAtDistanceFacingDirection(from,
-                                                                                                     overshoot/MGLMetersPerRadian,
+                                                                                                     overshoot/mbgl::util::M2PI * mbgl::util::EARTH_RADIUS_M,
```

Comment:
This was a paste mistake, also Gometry.swift implements it this [way](https://github.com/mapbox/mapbox-navigation-ios/blob/5be6ada250e43804d3d74c5724687a761abb6271/MapboxCoreNavigation/Geometry.swift#L90).